### PR TITLE
Bugs fixing, improvement and sample configs

### PR DESCRIPTION
**Pull request by tung-nt-niteco**

### CODE REVIEW

File: Dockerfile.7.0 [hunk 1]
```diff
@@ -22,10 +22,13 @@ RUN yum install -y centos-release-scl-rh && \
...
 
 # Install wordpress and backup the base image S2I scripts
-RUN cd /tmp && curl -o wordpress.tar.gz -SL https://wordpress.org/wordpress-${WORDPRESS_VERSION}.tar.gz \
+RUN echo "Install wordpress and backup the base image S2I scripts" \
+    && cd /tmp && curl -o wordpress.tar.gz -SL https://wordpress.org/wordpress-${WORDPRESS_VERSION}.tar.gz \
     && mkdir -p /opt/app-root/wordpress \
     && echo "$WORDPRESS_SHA1 *wordpress.tar.gz" | sha1sum -c - \
     && tar -xzf wordpress.tar.gz --strip-components=1 -C /opt/app-root/wordpress \
+	&& rm -rf /opt/app-root/wordpress/wp-content/plugins/akismet \
+	&& rm -f /opt/app-root/wordpress/wp-content/plugins/hello.php \
```
Comment (dukemai) [inline Dockerfile.7.0:31]: what does hello.php do? I am just curious to know

File: openshift/sites/bonniernews.se/wordpress-container_buildconfig_bn-bonniernews-test_5.0.2.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+kind: BuildConfig
+apiVersion: v1
+metadata:
+  annotations:
+    description: Build a container and push it to image streams
+  labels:
+    app: wordpress-tungtest
```
Comment (dukemai) [inline openshift/sites/bonniernews.se/wordpress-container_buildconfig_bn-bonniernews-test_5.0.2.yaml]: is this as intended?

File: Dockerfile.7.0 (hunk 1 above)
Comment (tung-nt-niteco) [reply Dockerfile.7.0:31]: https://wordpress.org/plugins/hello-dolly/
"When activated you will randomly see a lyric from Hello, Dolly in the upper right of your admin screen on every page."
The plugin was not activated and used in the Bonnier sites so we can clean up

File: openshift/sites/bonniernews.se/wordpress-container_buildconfig_bn-bonniernews-test_5.0.2.yaml (hunk 2 above)
Comment (tung-nt-niteco) [reply openshift/sites/bonniernews.se/wordpress-container_buildconfig_bn-bonniernews-test_5.0.2.yaml]: There are different ways to build and push the container image, this is one of the ways.
Or if the question is about WP 5.0.2, we need to patch WP core for different sites so yes we need to have one

Comment (dukemai) [reply openshift/sites/bonniernews.se/wordpress-container_buildconfig_bn-bonniernews-test_5.0.2.yaml]: I just wondered about the label, it sounds like a personal test on local environment

Comment (tung-nt-niteco) [reply openshift/sites/bonniernews.se/wordpress-container_buildconfig_bn-bonniernews-test_5.0.2.yaml]: Not really a test, the namespace can be changed.
And, if we do not have the one built for openshift namespace, this is the one done for a specific environment/project but taking the same s2i source. We have to patch the sites anyway.

File: Dockerfile.7.0 (hunk 1 above)
Comment (dukemai) [reply Dockerfile.7.0:31]: ok, thanks for your information

File: openshift/sites/bonniernews.se/wordpress-container_buildconfig_bn-bonniernews-test_5.0.2.yaml (hunk 2 above)
Comment (dukemai) [reply openshift/sites/bonniernews.se/wordpress-container_buildconfig_bn-bonniernews-test_5.0.2.yaml]: ok, thats good to know.

**Pull request closed without merging**